spec/support/helpers/model_properties.rb

```diff
@@ -51,7 +51,6 @@ module ModelProperties
     TRANSIENT_REGISTRATION_EXEMPTION = (REGISTRATION_EXEMPTION - %i[deregistration_message deregistered_at]).freeze
 
     REGISTRATION = %i[
-      reference
```

Comment:
I'm not sure we should be removing this from the attribute list – it means we are no longer testing Registration as completely in other places like https://github.com/DEFRA/waste-exemptions-engine/blob/0b872a0834ced4a95a75b41ea8fbf172e6bc8e82/spec/models/waste_exemptions_engine/registration_spec.rb#L10-L13